packages/hs-react-ui/src/components/Button/Button.tsx

```diff
@@ -130,35 +138,25 @@ const Button = ({
   const hasContent = Boolean(children);
   const { grayLight } = useColors();
   const containerColor = color || grayLight;
+  const props = {
+    'data-test-id': 'hsui-button',
+    id,
+    onClick,
+    onMouseDown,
+    onMouseUp,
+    elevation,
+    color: containerColor,
+    variant,
+    type,
+    disabled,
+  };
```

Comment:
Missing containerProps? Also probably needs a more descriptive name than just "props"

Reply:
You right